examples/blockly-rtc/src/http/index.js

```diff
@@ -23,31 +23,100 @@
  */
 
 import * as Blockly from 'blockly';
-import {getSnapshot, getEvents, writeEvents} from './workspace_client_handlers';
-import {getPositionUpdates, sendPositionUpdate} from './user_data_handlers';
+import { getSnapshot, getEvents, writeEvents } from './workspace_client_handlers';
+import { getPositionUpdates, sendPositionUpdate } from './user_data_handlers';
 import UserDataManager from '../UserDataManager';
 import WorkspaceClient from '../WorkspaceClient';
 
+const toolbar = {
+  kind: 'flyoutToolbox',
+  contents: [
+    {
+      kind: 'block',
+      type: 'controls_ifelse',
+    },
+    {
+      kind: 'block',
+      type: 'logic_compare',
+    },
+    {
+      kind: 'block',
+      type: 'logic_operation',
+    },
+    {
+      kind: 'block',
+      type: 'controls_repeat_ext',
+      inputs: {
+        TIMES: {
+          shadow: {
+            type: 'math_number',
+            fields: {
+              NUM: 10,
+            },
+          },
+        },
+      },
+    },
+    {
+      kind: 'block',
+      type: 'logic_operation',
+    },
+    {
+      kind: 'block',
+      type: 'logic_negate',
+    },
+    {
+      kind: 'block',
+      type: 'logic_boolean',
+    },
+    {
+      kind: 'block',
+      type: 'logic_null',
+      disabled: 'true',
+    },
+    {
+      kind: 'block',
+      type: 'logic_ternary',
+    },
+    {
+      kind: 'block',
+      type: 'text_charAt',
+      inputs: {
+        VALUE: {
+          block: {
+            type: 'variables_get',
+            fields: {
+              VAR: {
+                name: 'text',
+              }
+            },
+          },
+        },
+      },
+    }
+  ]
+}
+
 document.addEventListener('DOMContentLoaded', async () => {
   const workspace = Blockly.inject('blocklyDiv',
-      {
-        toolbox: document.getElementById('toolbox'),
-        media: 'media/',
-      });
+    {
+      toolbox: toolbar,
+      media: 'media/',
+    });
   const workspaceClient = new WorkspaceClient(
-      workspace.id, getSnapshot, getEvents, writeEvents);
+    workspace.id, getSnapshot, getEvents, writeEvents);
```

Comment:
Please revert this whitespace change (and similar). All line continuation should be indented by 4 spaces =)

Reply:
Sorry! What I meant is that this line actually needs to be indented /more/ not less hehe. Whenever you continue a line of code to a new line it needs to be indented by 4 spaces =)

Reply:
Done! 
Sorry for troubling 😅